### ISSUE TITLE
Renderer showing table and chart at the same time

### ISSUE THREAD

**Issue by diegobill** [nicolaskruchten/pivottable#1099]
Hi!

Do you know if it is possible configure a renderer that shows at the same time a table and a chart?

![image](https://user-images.githubusercontent.com/1781321/54867253-e670e880-4d5c-11e9-9c3e-43c5e6a01b90.png)

Thanks

**Comment by EricRossC** [comment]
I dont think so, but you can render two pivots, one configured to render as a chart and the other configured to render as a table, in the same page

![image](https://user-images.githubusercontent.com/812496/54867880-5898fb80-4d64-11e9-8162-eaf761c50a26.png)
